### PR TITLE
Revamp of the 'Team' page (now renamed 'Workgroups')

### LINKED ISSUE #895: Change "Developers" and "Contributors" pages into "Coordinators" and "Hall of Fame"
*This issue was created on gitlab and then migrated here. Only the original post was migrated, not the comments. Please take a look at the discussions on the original Gitlab issue before commenting here: https://repo.getmonero.org/monero-project/monero-site/-/issues/935*

---
I just had a literal showerthought. At the moment the team page is not very useful if somebody is looking for info about who to contact. I think the sections *Community* and *Developers* could be changed, in *Coordinators* and *Hall of fame*.

The *Coordinators* page would list all the coordinators of all the workgroups, with links and contacts, so that a visiotr immediatly know who to ask info to or where to look for more details.

The *Hall of Fame* should be a unic page including both *Community* and *Developers*. Basically the list of "individuals that have gone above and beyond for Monero". We could use this to push contributions or as a reward for particularly involved contributors, i don't know, just throw your ideas.

I think this way the *Team* page could be much more useful, right now it's basically just a list of names. Opinions about this?

### PR DESCRIPTION
**Pull request by erciccione**
Closes #895 

This changes completely the structure of the "Team" page, now renamed "Workgroups".

Instead of single members we show a list of the main workgroups, a description and brief instructions to start to contributing.

~~This PR is a draft because needs some minor edits and to give the opportunity to community member to improve descriptions and "where to start" sections~~. Feel free to suggest better wording, different instructions or new/different workgroups.

Preview:
Screenshot_2020-11-28  Workgroups

### CODE REVIEW

**Comment by xmrdsc** [comment]
Perhaps instead of `<i>` for the IRC channel names `<code>` looks better;

**Comment by SamsungGalaxyPlayer** [comment]
Not to get too meta here, but what will the Hangouts page turn into? What's the difference between that page and this one? If a workgroup makes a "Hangout" like a forum or an IRC channel, how will that be handled?

---
Monero Space

Monero Space is an active, project-focused Monero workgroup that provides services to the Monero community. It hosts popular resources and organizes events such as Monero Meet.

---

Malware Response Workgroup

The Monero Malware Response Workgroup provides resources and live support for multiple types of Monero-related malware.

#monero-mrw 

https://mrw.getmonero.org

**Comment by ThomasBucaioni** [comment]
> Perhaps instead of `<i>` for the IRC channel names `<code>` looks better;
Maybe just a Courier typeface?

**Comment by erciccione** [comment]
@SamsungGalaxyPlayer 

> but what will the Hangouts page turn into?

I consider the hangouts page as a simple list of chatrooms/places where community members can be found to discuss different topics. I see how the two can be seen as overlapping, but i don't think that's the case. For example `#monero-offtopic` makes sense in hangouts, but not in the workgroups page.

> workgroup makes a "Hangout" like a forum or an IRC channel, how will that be handled?

if it's a resource of the workgroup should be added in the entry of the workgroup (for eample `forum: link`), but could also be added to the hangouts page.

> Malware Response Workgroup

do you have a suggestion for the "Where to start" field?

**Comment by erciccione** [comment]
> Perhaps instead of <i> for the IRC channel names <code> looks better;

I considered it initially, but then avoided it because the highlighted text is slightly bigger on firefox:

Screenshot from 2020-10-20 15-25-07

but if people prefer it i'll just make the change.

**Comment by SamsungGalaxyPlayer** [comment]
@erciccione Where to start: Join the channel and ask questions or provide support.

**Comment by erciccione** [comment]
- Replaced `<i>` with `<code>`
- Edited description of monero space
- Added entry for "Malware Response Workgroup"
- Updated screenshot
- Marked 'Ready for review'

File: _i18n/en.yml [hunk 1]
```diff
@@ -124,14 +124,42 @@ sponsorships:
...
+  start: Where to start
+  contacts: Chatrooms and Contacts
+  chat: "Chat:"
+  website: "Website:"
+  core_descr: The members of this workgroup are always 7. They are the stewards of the community and they manage and maintain the Monero codebase and infrastructure. The Core Team act as mediators, weighting consensus in the community and taking decisions according to community's will.
+  core_start: The Core Team is a closed workgroup, it's not possible to join it unless invited.
+  community_descr: Workgroup used by the community to organize and coordinate. This is the place where every non-coder who wants to contribute to Monero should start.
+  community_start: The best way to start contributing to this workgroup is to join the chatrooms and introduce yourself, stating your interests and the skills you would like to offer to the community.
+  devworkgroup: Development Workgroup
+  devworkgroup_descr: This workgroup reunites Monero developers working on the wallets and daemon maintained by the Core Team. They are mostly divided in GUI and CLI devs and they coordinate using two different chatrooms.
```
Comment (SamsungGalaxyPlayer) [inline _i18n/en.yml:141]: "in GUI and CLI devs" is awkward wording. Maybe change the last sentence to "This workgorup consists of two main parts: the CLI and the GUI teams. These teams coordinate using two different chatrooms."

File: _i18n/en.yml [hunk 2]
```diff
@@ -124,14 +124,42 @@ sponsorships:
...
+  chatgeneral: "Chat (general):"
+  chatgui: "Chat (GUI):"
+  webworkgroup: Website Workgroup
+  web_descr: Another dev-focused workgroup where to coordinate the development of the website. Most devs in there are part of the development workgroup. The website workgroup was created mostly to have discussions related to the website all in one place.
+  web_start: It's always a good idea to join the chatroom and talk directly with the members of the workgroup, but if you prefer to get started by yourself, take a look at the readme in the GitHub repository. It contains step by step instructions to edit every single part of this website.
+  localization: Localization Workgroup
+  localization_descr: One of the oldest Monero Workgroups. It's formed by contributors working on translating Monero-related software and documentation. It's strongly suggested to contact the workgroup before working on translations.
```
Comment (SamsungGalaxyPlayer) [inline _i18n/en.yml:149]: Recommend removing first sentence, then changing the beginning of the second sentence to "This workgroup consists of contributors who translate..."

**Review by SamsungGalaxyPlayer — CHANGES_REQUESTED** [review]

File: _i18n/en.yml (hunk 2 above)
Comment (erciccione) [reply _i18n/en.yml:149]: Why removing the first sentence? it's an anecdotal reference, but at least makes the "description" field less monotone

**Comment by erciccione** [comment]
Rebased and added suggestions.

File: _i18n/en.yml [hunk 3]
```diff
@@ -141,14 +140,42 @@ sponsorships:
...
+  core: Core Team
+  community: Community Workgroup
   mrl: Research Lab
-  thanks: Special Thanks
+  descr: Description
+  start: Where to start
+  contacts: Chatrooms and Contacts
+  chat: "Chat:"
+  website: "Website:"
+  core_descr: The members of this workgroup are always 7. They are the stewards of the community and they manage and maintain the Monero codebase and infrastructure. The Core Team act as mediators, weighting consensus in the community and taking decisions according to community's will.
```
Comment (binaryFate) [inline _i18n/en.yml:152]: core_descr: Stewards of the community, the Core Team manages and maintains infrastructure where centralization cannot be avoided (domains, website, github repositories...). Its members act as mediators as they see fit and aim to weigh consensus in the community, perpetuating the quality interactions that made Monero the successful project it is today. The Core Team does not and has never received any financial compensation.

**Comment by binaryFate** [comment]
Suggested new text for core team. (Agreed with the others).

File: _i18n/en.yml (hunk 3 above)
Comment (binaryFate) [reply _i18n/en.yml:152]: Note this text was agreed with the others

Comment (erciccione) [reply _i18n/en.yml]: Updated

File: _i18n/en.yml (hunk 2 above)
Comment (SamsungGalaxyPlayer) [reply _i18n/en.yml:165]: Just because it doesn't describe anything relevant/actionable to the audience imo.

Comment (erciccione) [reply _i18n/en.yml:165]: I disagree. The localization workgroup is the oldest organized workgroup of the community. IMO it's relevant to mention it in a page dedicated to Monero workgroups and it's also something that makes me proud. It's a description, doesn't need to be actionable.

**Comment by binaryFate** [comment]
@erciccione shall we proceed or are you waiting for some more feedback? I think it's a cool PR I want to see it out :)

**Comment by erciccione** [comment]
@binaryFate i'm waiting for reviews/approvals :)

**Comment by SamsungGalaxyPlayer** [comment]
Would it be possible to add the following for Monero Space: 

Matrix: https://matrix.to/#/#monero-space:matrix.org

Flarum: https://forum.monero.space (with the [Flarum logo?](https://opencollective-production.s3.us-west-1.amazonaws.com/0959f4e0-0559-11ea-97cf-5fc2f9568174.png))

Website: https://monero.space

**Comment by erciccione** [comment]
> Would it be possible to add the following for Monero Space:

Added (except the Flarum logo) and updated the preview

**Review by SamsungGalaxyPlayer — APPROVED** [review]

File: _i18n/en.yml [hunk 4]
```diff
@@ -141,14 +140,42 @@ sponsorships:
...
+  localization_start: Contact the workgroup on the chatroom or open an issue on the GitHub repository. You can also start translating immediately using the Weblate platform. Just make sure to read the guide for translators first.
+  locplatform: "Localization platform:"
+  outreach_descr: This workgroup is focused on getting Monero out there, gathering artists, writers and anyone who wish to make Monero known to the World.
+  outreach_start: The members of the Outreach workgroup heavily use Telegram for coordination and matrix/freenode in minor part. If you want to help, you have to contact the workgroup on one of this channels. They are quite structured, so you'll probably be directed to a coordinator.
+  telegram: "Telegram chat:"
+  mrl_descr: As the name suggests, the Monero Research Lab is the workgroup focused on Monero related research. It's formed by scientists trying to find a way to improve Monero's protocol and features. Most of the protocol improvements who make Monero amazing, like CLSAG and @Bulletproofs, are the result of the work of this workgroup.
```
Comment (hyc) [inline _i18n/en.yml:171]: I know this PR has already been merged, but this should have been:
s/find a way/find ways/   <-- are we really only looking for one way to improve? nope.
s/improvements who make/improvements that make/   <-- the improvements are things, not people.

Comment (erciccione) [reply _i18n/en.yml:171]: #1378

**Pull request merged**